languages/julia/exercises/concept/poetry-club/.meta/test_generator_util.jl

```diff
@@ -0,0 +1,249 @@
+# Writing out the tests explicitely avoids giving away the solution in the testsuite.
+# It also leads to a minimal number of prerequisites.
+# This file can be used to generate them.
+
+const tab = "    "
+
+include("example.jl")
+
+struct Poem
+    title::String
+    body::String
+    author::String
+    acrostic::Bool
+    telestich::Bool
+end
+
+const poems = (
+    Poem("Shire",
+    
+        """
+        Stands so high
+        Huge hooves too
+        Impatiently waits for
+        Reins and harness
+        Eager to leave""",
+
+        "Michael Lockwood",
+        true,
+        true
+    ),
+
+    Poem("Summer",
+        
+        """
+        Sunshine warming my toes,
+        Underwater fun with my friends.
+        Making homemade ice cream on the porch,
+        Many long nights catching fireflies.
+        Early morning walks to the creek,
+        Reveling in the freedom of lazy days.""",
+
+        "John Albert Caballero",
+        true,
+        false
+    ),
+
+    Poem("Sophia",
+
+        """
+        Serene, calming quality
+        Organized, you always have it together
+        Picturesque, strikingly beautiful
+        Honest, so genuine
+        Imaginative, a creative mind
+        Alluring, so attractive""",
+
+        "John Albert Caballero",
+        true,
+        false
+    ),
+
+    Poem("Code",
+
+        """
+        Compilers intensily bestow
+        On commencing without ego
+        Different processes ajar
+        Exit with zero quick""",
+
+        "Derk-Jan Karrenbeld",
+        true,
+        true
+    ),
+
+    Poem("",
+    
+        """
+        Ιησούς
+        Χριστός
+        Θεού
+        Υἱός
+        Σωτήρ""",
+
+        "",
+        true,
+        false
+    ),
```

Comment:
I would prefer not using a religious poem here, however it's quite hard to find poems that work in a language one doesn't speak. Let's leave it as placeholder and replace later if/when we found something.

---

From https://en.wikipedia.org/wiki/Acrostic and https://de.wikipedia.org/wiki/Akrostichon

Reply:
I can write one in Dutch or find one in French, if that helps.

Reply:
For this particular test case I needed something with non-latin letters, so it wouldn't help. But feel free if you want to anyway